configure.cmd

```diff
@@ -72,7 +71,7 @@ if defined ROS_ARCH (
     cl 2>&1 | find "19.12." > NUL && set VS_VERSION=15
     cl 2>&1 | find "19.13." > NUL && set VS_VERSION=15
     if not defined VS_VERSION (
-        echo Error: Visual Studio version too old or version detection failed.
+        echo Error: Visual Studio version too old ^(before 10 ^(2010^)^) or version detection failed.
```

Comment:
why?

Reply:
To be explicit, to give a clue on which case is happening.